app/assets/javascripts/claims-table.js

```diff
@@ -29,7 +29,12 @@ $(document).ready(function () {
     order: [],
     ajax: {
       url: dataReference,
-      dataSrc: 'claims'
+      dataSrc: 'claims',
+      error: function (response) {
+        var error = JSON.parse(response.responseText)
+        $('#claims_processing').hide()
+        alert('An error occurred when searching for claims. ' + error.name + ': ' + error.message) // eslint-disable-line no-undef
```

Comment:
How about `when retrieving claims` instead of `when searching for claims` in this case since it is the list view and not a search page?